tests/seq_io/test_genbank.gbk

```diff
@@ -0,0 +1,90 @@
+LOCUS       SCU49845     5028 bp    DNA             PLN       21-JUN-1999
```

Comment:
is there no other genbank file anywhere that you can use without adding a new one? If not, it is ok to add a new one!